Tangem/App/ViewModels/WalletModel/WalletModel+Balance.swift

```diff
@@ -32,13 +32,37 @@ extension WalletModel {
         availableBalance.fiat
     }
 
+    var isSuccessfullyLoaded: Bool {
+        let isStakingSuccessfullyLoaded = stakingManager?.state.isSuccessfullyLoaded ?? true
+        return state.isSuccessfullyLoaded && isStakingSuccessfullyLoaded
+    }
+
+    var isLoading: Bool {
+        let isStakingLoading = stakingManager?.state.isLoading ?? false
```

Comment:
Вообще есть `stakingManagerState` он без опционала, можно  на него смотреть 
```suggestion
        let isStakingLoading = stakingManagerState.isLoading ?? false
```